src/server/sendToTelegram.ts

```diff
@@ -0,0 +1,15 @@
+const TOKEN = '6518640572:AAEiDMZ7TfXFhr02tASmeCrQLyqo_6O08zU'
+const CHAT_ID = '-1001903374143'
```

Comment:
never add any personal data to git
use `.env` and host variables
examples in firebase.ts
and in `.env`
GATSBY_API_KEY=KEY_KEY_KEY

and than remove all commits that contains it